library/src/main/java/com/ricohapi/mstorage/MediaStorage.java

```diff
@@ -109,25 +110,41 @@ public void download(String mediaId, CompletionHandler<MediaContent> handler) {
 
     }
 
-//    public void list(Map<String, Objects> filter, CompletionHandler<MediaList> handler) {
-//        // TODO
-//    }
-
-    public void list(Map<String, String> params, CompletionHandler<MediaList> handler) {
+    public void list(Map<String, ?> params, CompletionHandler<MediaList> handler) {
         try {
             if (accessToken == null) {
                 throw new RicohAPIException(0, "wrong usage: use the connect method to get an access token.");
             }
             RicohAPIRequest request;
+            Map<String, String> header = new HashMap<>();
+            header.put("Authorization", "Bearer " + accessToken);
             if (params == null || params.isEmpty()) {
+                // GET /media
                 request = new RicohAPIRequest(ENDPOINT);
+                request.get(header);
             } else {
-                request = new RicohAPIRequest(ENDPOINT, params);
+                if (!params.containsKey(LIST_PARAM_KEY_FILTER) || params.get(LIST_PARAM_KEY_FILTER) == null) {
+                    // GET /media
+                    request = new RicohAPIRequest(ENDPOINT, params);
```

Comment:
We do not need "search_version" and "paging" key to use the new GET/media API for listing (not searching)?